tests/futil_test.py

```diff
@@ -0,0 +1,48 @@
+# coding=utf-8
+
+import unittest
+import os
+
+import ladybug.futil as futil
+
+
+class DataCollectionTestCase(unittest.TestCase):
+    """Test for (ladybug/futil.py)"""
+
+    # preparing to test.
+    def setUp(self):
+        """set up."""
+
+    def tearDown(self):
+        """Nothing to tear down as nothing gets written to file."""
+        pass
+
+    def test_download_file(self):
+        """Test the download file capability"""
+        wf_url = "https://www.energyplus.net/weather-download/" \
+            "southwest_pacific_wmo_region_5/AUS//AUS_NSW.Sydney.947670_IWEC/all"
+        wf_name = os.path.join("./tests/zip", "AUS_NSW.Sydney.947670_IWEC.zip")
+        futil.download_file(wf_url, wf_name)
+        assert os.path.isfile(wf_name)
```

Comment:
should you check approximate size? It can download an empty file.

Reply:
Good idea.  Done!